Microsoft.UpgradeAssistant.Mappings.sln

```diff
@@ -0,0 +1,37 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.9.34701.34
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.UpgradeAssistant.Mappings", "src\Microsoft.UpgradeAssistant.Mappings\Microsoft.UpgradeAssistant.Mappings.csproj", "{BC5DCE4F-9D8F-4DC2-94C1-8DFCD58C7CA1}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.UpgradeAssistant.Mappings.Tests", "tests\Microsoft.UpgradeAssistant.Mappings.Tests\Microsoft.UpgradeAssistant.Mappings.Tests.csproj", "{7B5CF532-A3AD-4375-8B15-7A6B63A1B00F}"
```

Comment:
>Microsoft [](http://example.com/codeflow?start=53&length=9)

solution file would be reused for other projects too? Maybe name it UpgradeAssistant.Extensions.sln?